docker/sds-api/spine-directory-service/sds/request/routing_reliability_handler.py

```diff
@@ -33,5 +35,8 @@ async def get(self):
         logger.info("Combined routing and reliability information. {routing_reliability_information}",
                     fparams={"routing_reliability_information": combined_info})
 
-        self.write(combined_info)
+        if content_type == 'application/fhir+xml':
+            self.write(get_xml_format(combined_info, org_code, service_id))
+        else:
```

Comment:
should be `elif content_type == 'application/fhir+json'` and then `else` throwing exception

Reply:
the default response is fhir+json. Validation is done at line 18 'get_valid_accept_type'